doc/onboarding.md

```diff
@@ -86,6 +86,13 @@ onboarding session.
       `semver-major` label
     * When adding a `semver-*` label, add a comment explaining why you're adding
       it. Do it right away so you don't forget!
+  * Please add the `ready` label for PRs where:
+    * the CI has been started,
```

Comment:
The CI has been successfully run.

Reply:
If CI was required, as some of this documentation indicates it may not be.